CODEOWNERS

```
@@ -0,0 +1,4 @@
+@hashicorp/vault
+@hashicorp/terraform
+@hashicorp/consul
+@hashicorp/nomad
```

Thanks for adding this. `nomad-eng` is the correct team to use on GitHub from the Nomad side.
```suggestion
@hashicorp/nomad-eng 
```